### PR TITLE
Use startupservice and persistence services of toolkit

### PR DESCRIPTION
**Pull request by MohamedHamouGisaia**

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -99,7 +101,9 @@
               "src/assets",
               "src/default.json",
               "src/config.json",
-              "src/env.js"
+              "src/env.js",
```
Comment (mbarbet) [inline angular.json:104]: do we still need this? env.js ?

File: package.json [hunk 2]
```diff
@@ -28,11 +28,9 @@
     "@angular/platform-browser-dynamic": "~8.2.10",
     "@angular/router": "~8.2.10",
     "@types/file-saver": "^2.0.1",
-    "@ngx-translate/core": "^12.1.2",
-    "@ngx-translate/http-loader": "^4.0.0",
     "ajv": "^6.4.0",
     "ajv-keywords": "^3.4.0",
-    "arlas-wui-toolkit": "13.0.0",
+    "arlas-wui-toolkit": "14.0.0-beta.5",
     "arlas-persistence-api": "13.0.0-beta.2",
```
Comment (mbarbet) [inline package.json:34]: we can remove arlas-persistence-api import ?

File: angular.json (hunk 1 above)
Comment (MohamedHamouGisaia) [reply angular.json:104]: no I should remove it

File: package.json (hunk 2 above)
Comment (MohamedHamouGisaia) [reply package.json:34]: yes totally

**Pull request merged**